prometheus/docker-compose.yaml

```diff
@@ -11,10 +11,16 @@ services:
       context: config
       args:
         PROMETHEUS_CONFIG_YTT_VERSION: ${PROMETHEUS_CONFIG_YTT_VERSION}
+    profiles:
```

Comment:
I think config shouldnt have profiles, because it should always run?

Reply:
It looks like only the prometheus and alertmanager services depend on config, so my thinking is that if you want to install only node-exporter and/or cadvisor (without prometheus or grafana), then you don't need config. Is that not a good way to go?

Reply:
Ok so those dont need any config, makes sense now